packages/test/generatedCode/models/serializeAttachmentCollectionResponse.ts

```diff
@@ -3,6 +3,17 @@ import {serializeAttachment} from './serializeAttachment';
 import {AdditionalDataHolder, DeserializeIntoModelFunction, Parsable, ParseNode, SerializationWriter} from '@microsoft/kiota-abstractions';
 
 export function serializeAttachmentCollectionResponse(writer: SerializationWriter, attachmentCollectionResponse: AttachmentCollectionResponse | undefined = {}) : void {
-            writer.writeStringValue("@odata.nextLink", attachmentCollectionResponse.odataNextLink);
-            writer.writeCollectionOfObjectValues<Attachment>("value", attachmentCollectionResponse.value, serializeAttachment);
+        for (const [key, value] of Object.entries(attachmentCollectionResponse)){
+            switch(key){
+                case "@odata.nextLink":
```

Comment:
this should be the symbol name and not the serialization name being used here

Reply:
done